docker/scripts/functions.sh

```diff
@@ -106,14 +106,15 @@ function configure_image_cache_dns() {
 	cp -a /etc/hosts /etc/hosts.new
 	{
 		echo "$images_ip        github.com"
+		echo "$images_ip        github-cloud.githubusercontent.com"
 		echo "$images_ip        github-cloud.s3.amazonaws.com"
 		echo "$images_ip        github-mirror.packet.net"
 	} >>/etc/hosts.new
 	# Note: using mv here fails (415 Unsupported Media Type) because docker sets
 	# this up as a bind mount and we can't replace it.
 	cp -f /etc/hosts.new /etc/hosts
 	echo -n "LFS pulls via github-cloud will now resolve to image cache:"
-	getent hosts github-cloud.s3.amazonaws.com | awk '{print $1}'
```

Comment:
I may be misreading the PR description, but I feel like this line removal feels counter to it:

`It's not yet confirmed whether they've stopped using the AWS s3 hostname, so we're leaving that in place for the time being.`

Reply:
This particular line is just printing what the IP is that github-cloud.s3.amazonaws.com/github-cloud.githubusercontent.com is currently pointing to. Just above this, we hard-wire this to use `$images_ip` in /etc/hosts, so all this is doing is printing a verification that this hostname is resolving to our anycast IP for IPN.